tests/test_routes.py

```diff
@@ -81,18 +90,6 @@ def test_create_inventory_records(self):
         self.assertEqual(new_record["reorder_quantity"], test_record.reorder_quantity)
         self.assertEqual(new_record["restock_level"], test_record.restock_level)
 
-        #uncomment this once list all products works
```

Comment:
actually this was used to check the location header is correct, which could have only been done if we could return the record with location properties. we can leave this commented until we have a function that gets data from the database using a filter

Reply:
Reverted the change!

Reply:
Thanks!